doc/api.rst

```diff
@@ -4,6 +4,34 @@ API Documentation
 -----------------
 The primary method of using RSMTool is via the command-line scripts :ref:`rsmtool <usage_rsmtool>`, :ref:`rsmeval <usage_rsmeval>`, :ref:`rsmpredict <usage_rsmpredict>`, :ref:`rsmcompare <usage_rsmcompare>`, and :ref:`rsmsummarize <usage_rsmsummarize>`. However, there are certain functions in the ``rsmtool`` API that may also be useful to advanced users for use directly in their Python code. We document these functions below.
 
+.. note::
+    
+     RSMTool v5.7 and older provided the API functions ``metrics_helper``, ``convert_ipynb_to_html``, and ``remove_outliers``. These functions have now been turned into static methods for different classes. If you are using these functions in your code and want to migrate to the new API, you should replace the follwing statements in your code:
+
+    .. code-block:: python
+
+        from rsmtool.analysis import metrics_helper
+        metrics_helper(...)
+
+        from rsmtool.report import convert_ipynb_to_html
+        convert_ipynb_to_html(...)
+        
+        from rsmtool.preprocess import remove_outliers
+        remove_outliers(...)
+
+    with the following, respectively:
+
+    .. code-block:: python
+
+        from rsmtool.analyzer import Analyzer
+        Analyzer.metrics_helper(...)
+
+        from rsmtool.reporter import Reporter
+        Reporter.convert_ipynb_to_htmlfr(...)
```

Comment:
Should be `Reporter.convert_ipynb_to_html(...)`, right?

Reply:
Ugh. Yes :)